core/server/helm_release_test.go

```diff
@@ -138,6 +138,84 @@ func TestGetHelmRelease(t *testing.T) {
 	g.Expect(err).To(HaveOccurred())
 }
 
+/**
+ * This test demonstrates the behavior of ListHelmReleases.
+ * The third HR created in this test is managed by the Helm Release Controller,
+ * but does not have a corresponding secret. Because of this the getHelmReleaseInventory
+ * call throws an error. This should not cause the entire list call to error out, and
+ * instead skip just the one HR that is in error state.
+ **/
+func TestListHelmReleases_withInventory(t *testing.T) {
+	g := NewGomegaWithT(t)
+	ctx := context.Background()
+
+	c, _ := makeGRPCServer(k8sEnv.Rest, t)
+
+	scheme, err := kube.CreateScheme()
+	g.Expect(err).To(BeNil())
+
+	k, err := client.New(k8sEnv.Rest, client.Options{
+		Scheme: scheme,
+	})
+	g.Expect(err).NotTo(HaveOccurred())
+
+	appName := "myapp" + rand.String(5)
+	nsA := newNamespace(ctx, k, g)
+	nsB := newNamespace(ctx, k, g)
+	nsC := newNamespace(ctx, k, g)
+
+	newHelmRelease(ctx, appName, nsA.Name, k, g)
+	newHelmRelease(ctx, appName, nsB.Name, k, g)
+
+	res1, error1 := c.ListHelmReleases(ctx, &pb.ListHelmReleasesRequest{})
+
+	g.Expect(error1).NotTo(HaveOccurred())
+
+	releasesFound := len(res1.HelmReleases)
```

Comment:
There was an error on this line, before I added a newline between it and the `g.Expect` before it. Curiously, this was not caught by `make fmt`.